examples/analog_out/voltage_update.py

```diff
@@ -0,0 +1,12 @@
+"""Example of analog output voltage generation.
+
+This example demonstrates how to output a single Voltage Update
+(Sample) to an Analog Output Channel.
+"""
+
+import nidaqmx
+
+with nidaqmx.Task() as task:
+    task.ao_channels.add_ao_voltage_chan("Dev1/ao0")
+
+    print(f"Generate {task.write(1.1)} voltage sample.")
```

Comment:
Change "generate" to either "generating" or "generated" depending on whether it's before or after the call to task.write.

Reply:
Noted, it's updated.